src/renderers/shaders/ShaderChunk/common.glsl

```diff
@@ -4,6 +4,10 @@
 #define LOG2 1.442695
 #define EPSILON 1e-6
 
+#define square(a) a * a
```

Comment:
Easier to manually hoist and do the multiply then. Added brackets for whiteCompliment